examples/server/webui/src/main.js

```diff
@@ -75,6 +83,8 @@ const THEMES = ['light', 'dark', 'cupcake', 'bumblebee', 'emerald', 'corporate',
 const VueMarkdown = defineComponent(
   (props) => {
     const md = shallowRef(new MarkdownIt({ breaks: true }));
+    md.value.use(HighlightJS, { hljs });
+    md.value.use(markdownItKatexGpt)
```

Comment:
again, for trivial changes like this, you should pay more attention to the coding style.

it's not fun when your code is just 10 lines and I ended up leaving 3 comments

```suggestion
    md.value.use(markdownItKatexGpt);
```

Reply:
Thanks for pointing these out. I'll be more careful next time. I appreciate you taking the time to do this. 
Please let me know if there's anything else I can fix or work on. I'm looking for feedback.

It might be a good idea to add a code formatter and linter in the CI or/and pre-commit to prevent this from happening in the future.